generated/nidmm/nidmm.proto

```diff
@@ -119,7 +119,6 @@ enum NiDmmAttribute {
   NIDMM_ATTRIBUTE_SPECIFIC_DRIVER_PREFIX = 1050302;
   NIDMM_ATTRIBUTE_IO_RESOURCE_DESCRIPTOR = 1050304;
   NIDMM_ATTRIBUTE_LOGICAL_NAME = 1050305;
-  NIDMM_ATTRIBUTE_IO_SESSION = 1050322;
```

Comment:
`NIDMM_ATTRIBUTE_IO_SESSION`, `NIDMM_ATTRIBUTE_CLASS_DRIVER_CLASS_SPEC_MAJOR_VERSION `, and `NIDMM_ATTRIBUTE_CLASS_DRIVER_CLASS_SPEC_MINOR_VERSION` are now missing.

But these are from [IviDmmObsolete.h](https://github.com/ni/grpc-device/blob/main/imports/include/IviDmmObsolete.h). Should these be included in the enum or should we just remove them?

Reply:
Marcos said in the ni-central PR: "I don't think IVIDMM_ATTR_IO_SESSION means anything in NI-DMM. I don't even know what it returns. It's for IVI drivers implemented on top of VISA iirc.

The other two I suppose are harmless."

I'm still trying to figure out what to do with these. These variants aren't in the other drivers so I think they were added erroneously here. I guess the question here is if we want / need to keep them for backwards compatibility or not.

Reply:
Let's leave them out. 
Same reasoning as before - if someone was using it, we break it in the next release & lead them to the correct way (of not using it)

Reply:
Updated so that method and attributes are no longer generated.